src/_locales/en/messages.json

```diff
@@ -123,6 +123,14 @@
     "message": "Show notifications.",
     "description": "Show notifications before an event starts."
   },
+  "show_add_to_google_calendar_in_context_menu": {
+    "message": "Show 'Add to Google Calendar' in context menu",
+    "description": "Add the 'Add to Google Calendar' action in context menu"    
+  },
+  "other_options": {
```

Comment:
Instead of a new section, let’s just remove the Toolbar Options as a separate setting group, and merge them all together.